python/secure_install/pipeline_scripts/create_install_archive.bash

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This script gathers AaC's runtime dependencies, creates a hash file, pulls in install scripts, and generally
+#   prepares the secure installation file. We'll delegate to Github Action's automatic zipping of artifacts to
+#   ultimately compress the directory into an archive.
+
+install_script_dir=$(dirname "$0")
+cd $install_script_dir
+cd ../
+
+version=$(aac version)
+install_dir="aac_secure_install_$version"
+
+[[ -d $install_dir ]] && rm -r $install_dir
+mkdir $install_dir
+cp -p -r install_scripts/*.bash $install_dir
+cp README.md $install_dir
+cd $install_dir
+
+pip wheel ../../
+pip-compile --generate-hashes ../../setup.py
```

Comment:
I guess this may be where the `requirements.txt` file comes from? If so, shouldn't we instruct users to run this script before the other ones? Or it looks like we could run it in the pipeline and make the file a pipeline artifact.

Reply:
This script is only intended to be used in the pipeline.